rafay/resource_resourcetemplate.go

```diff
@@ -1271,7 +1287,7 @@ func flattenResourceTemplateSpec(in *eaaspb.ResourceTemplateSpec, p []interface{
 
 	obj["agents"] = flattenEaasAgents(in.Agents)
 	obj["sharing"] = flattenSharingSpec(in.Sharing)
-	obj["artifact_driver"] = flattenWorkflowHandlerCompoundRef(in.ArtifactDriver)
```

Comment:
since we are deprecating and not stopping support completely, we should continue to have support for artifact_driver as of now.